### ISSUE TITLE
SO_ terms not being expanded in pdb e.g. gene SO_0000704

### ISSUE THREAD

**Issue by Robbie1977** [VirtualFlyBrain/VFB_neo4j#97]
see http://pdb.virtualflybrain.org/browser/
```MATCH (n:Class) where n.short_form starts with 'SO_' RETURN n LIMIT 25```

https://v2.virtualflybrain.org/?i=VFB_00017894&id=FBgn0028374

**Comment by Robbie1977** [comment]
@dosumis we now have a gene node from S0 but ols has loaded both ours and the SO one with the same short_form. This maybe fine but would be better if they were merged?

See:
http://pdb-firstload.virtualflybrain.org/browser/
```MATCH (n:Class {short_form:'SO_0000704'}) RETURN n LIMIT 25```

Also SO_0000400

**Comment by dosumis** [comment]
> @dosumis <https://github.com/dosumis> we now have a gene node from S0 but ols has loaded both ours and the SO one with the same short_form. This maybe fine but would be better if they were merged?
> 
> See:
> http://pdb-firstload.virtualflybrain.org/browser/ <http://pdb-firstload.virtualflybrain.org/browser/>
> MATCH (n:Class {short_form:'SO_0000704'}) RETURN n LIMIT 25
> 
> Also SO_0000400
> 

Yep.  That’s what OLS does - the same term loaded from different files ends up as different nodes. V2 pipeline pdb will fix this long term, but in the short term we can fix it by merging everything we need into vfb.owl before loading.  This should be pretty straightforward with some edits to the VFB.owl release pipeline - adding a couple of more steps with owltools or Robot.
